src/xpk/commands/workload.py

```diff
@@ -327,8 +327,7 @@ def workload_create(args) -> None:
 
   if args.headless and not is_cluster_using_clouddns(args):
     xpk_print(
-        'Please run xpk cluster create-pathways first, to upgrade and enable'
-        ' CloudDNS on your cluster.'
+        'Please use kubectl port forwarding to connect to the Pathways proxy.'
     )
     xpk_exit(1)
```

Comment:
Any reason to continue to exit here? Can we just get rid of this check and have the port forwarding instructions printed out when starting headless workload?

```
kubectl port-forward pod/<proxy-pod-name> <proxy-port>:<proxy-port>
JAX_PLATFORMS=proxy JAX_BACKEND_TARGET=grpc://127.0.0.1:<proxy-port> python -c 'import pathwaysutils; import jax; print(jax.devices())'
```

Reply:
Removed the stale exit and also added instructions for port forwarding.
Removed all clouddns checks (not needed for existing clouddns enabled clusters also.)
Thanks.